apps/e2e/cypress/support/e2e.js

```diff
@@ -27,7 +27,7 @@ after(() => {
 
 after(() => {
 	//cy.clearAllSessionStorage();
-	cy.clearCookies();
+	//cy.clearCookies();
```

Comment:
do you need this code anymore?

Reply:
May in future...Just want to check how tests are going to behave without clearing the cookies. If tests are stable I can remove in next release